ipython/mechanism_analyzer.ipynb

```diff
@@ -17,7 +17,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
```

Comment:
please clean up the output, usually we don't include the output of ipython notebook on master